ci-operator/step-registry/ipi/conf/vsphere/zones/multisubnets/ipi-conf-vsphere-zones-multisubnets-commands.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# ensure LEASED_RESOURCE is set
+if [[ -z "${LEASED_RESOURCE}" ]]; then
+  echo "Failed to acquire lease"
+  exit 1
+fi
+
+echo "$(date -u --rfc-3339=seconds) - sourcing context from vsphere_context.sh..."
+# shellcheck source=/dev/null
+declare vsphere_datacenter
+declare vsphere_url
+source "${SHARED_DIR}/vsphere_context.sh"
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/govc.sh"
+
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+base_domain=$(<"${SHARED_DIR}"/basedomain.txt)
+machine_cidr=$(<"${SHARED_DIR}"/machinecidr.txt)
+
+
+cat >> "${CONFIG}" << EOF
+baseDomain: $base_domain
+controlPlane:
+  name: "master"
+  replicas: 3
+  platform:
+    vsphere:
+      zones:
+       - "us-east-1"
+       - "us-east-2"
+       - "us-east-3"
+compute:
+- name: "worker"
+  replicas: 3
+  platform:
+    vsphere:
+      zones:
+       - "us-west-1"
```

Comment:
The multisubnets is only for controlPlane ?

Reply:
yes, refer to https://issues.redhat.com/browse/OCPSTRAT-462.

Reply:
if we also place workers in multisubnets, what would happen? Does it work?

Reply:
It has not been verified yet, if we need to support worker in multisubnets, I guess we also need apply more subnets like ocp-ci-seg-20 and ocp-ci-seg-21.

Reply:
I think we do not need more subnets for workers, we can re-use the same subnets for workers, but before that, we need to identify what "multipsubnets" does mean, does it mean masters and workers are placed into diffferent subnets, or we just need to ensure each master in different subnets. 

Personally we can place each worker in different subnets together in this PR, if we do not, I would suggest to rename the step name to `ipi-conf-vsphere-zones-multisubnets-masters` to indicate the step is only for masters.